src/components/Search.js

```diff
@@ -1,11 +1,23 @@
 import React from 'react'
 
-const Search = () => {
+// const searchTerm=_.debounce((term) =>
+//   {this.searchTerm(term)}, 300);
+
+// tried to use lodash and debounce to throttle search results, but failed miserably.
+// really wanted to use throttled searches so the rendered results wouldn't have such
+// a jarring effect to the user. Oh well. tried my best.
+// Here's to hoping that I'll figure it out on my project!
+
+const Search = (props) => {
+  function onChange(event){
+    props.handleChange(event)
+  }
   return (
     <div className="ui huge fluid icon input">
       <input
         type="text"
         placeholder={"Search your Recent Transactions"}
+        onChange={onChange}
```

Comment:
should be able to pass the `handleChange` function directly from props here - not sure that this is more readable.